sphinxcontrib/rst2myst/builders/myst.py

```diff
@@ -100,7 +100,7 @@ def copy_static_files(self):
                 entry = path.join(self.confdir, static_path)
                 copy_asset(entry, output_path)
         else:
-            logger.warning("rst2myst_static_file_path not specified in conf.py")
+            logger.info("rst2myst_static_file_path not specified in conf.py")
```

Comment:
@AakashGfude is this config setting `required`? Can we just use a default value.

Reply:
hmm. what default entry should we specify? the one specified in `add_config_value` is empty array.  Put it here, just in case, we forget to specify it in conf.py and are thinking why is the `static` folder not passing through.

Reply:
most projects shouldn't need to specify this right?  Unless _static folder is in a location outside of the typical locations in the `source` folder or at the root level of a `sphinx` project.

Reply:
this is the default in `sphinx` for `html_file_path`?

```python
html_static_path = ['_static']
```

Reply:
Perhaps we should use:

```python
rst2myst_static_path = ['_static']
```

Reply:
hmm. this sounds right.